.travis.yml

```diff
@@ -1,6 +1,13 @@
 language: node_js
 before_install:
   - npm install -g npm@latest
+install:
+  - npm install --global electron
```

Comment:
Can we also make `electron` a development dependency and re-use that instead? That will give us a unified way to control in which electron version this code runs on.